mindmeld/app.py

```diff
@@ -161,6 +168,97 @@ def add_dialogue_rule(self, name, handler, **kwargs):
         else:
             self._dialogue_rules.append((name, handler, kwargs))
 
+    def custom_action(self, **kwargs):
+        """Adds a custom action, similar to `add_custom_action` but allows the ordering
+        of nlp entities to be more flexible.
+
+        Examples:
+            app.custom_action(intent='greeting', action='say_greeting')
+            app.custom_action(entity='person', action='greet_person')
+        """
+        action = kwargs.pop("action", None)
+        actions = kwargs.pop("actions", None)
```

Comment:
Why cant we just use one field called actions where it can be a list of 1 or many?

Reply:
It is not grammatically correct since `action` is singular. It is also consistent with how we use the `handle` function (`entity` vs `entities`)